src/main/java/calculator/Calculator.java

```diff
@@ -0,0 +1,72 @@
+package calculator;
+
+import calculator.operator.Operator;
+import calculator.operator.OperatorFactory;
+
+import java.util.Arrays;
+
+public class Calculator {
+
+    private final static String FORMULA_SEPARATION_KEYWROD = " ";
+    private final static String[] OPERATOR_KEWORDS = {"+", "-", "*", "/"};
+
+    private String[] formula;
+
+    public Calculator(String input) {
+        validationEmptyWithInput(input);
+        this.formula = validationInvalidFormulaWithInput(input);
+    }
+
+    public Double calculate() {
+        double beforeNum = Double.parseDouble(formula[0]);
+
+        for (int i = 1; i < this.formula.length - 1; i=i+2) {
```

Comment:
```suggestion
        for (int i = 1; i < this.formula.length - 1; i = i + 2) {
```

공백 컨벤션 위반입니다. IDE에서 컨벤션을 자동으로 맞춰주는 shortcut이 있으니 확인해보세요.
intelliJ 는 option + command + l 입니다.

8.2 blank spaces 항목을 확인해보세요.
https://myeonguni.tistory.com/1596